ui/app/src/Utils/Views.elm

```diff
@@ -22,7 +22,9 @@ labelButton maybeMsg labelText =
     case maybeMsg of
         Nothing ->
             span
-                [ class "btn btn-sm bg-faded btn-secondary mr-2 mb-2" ]
+                [ style [ ( "user-select", "text" ) ]
```

Comment:
This works in chrome, but not firefox. Check out https://github.com/prometheus/alertmanager/blob/master/ui/app/src/Views/AlertList/AlertView.elm#L98-L102.